build/requirements.txt

```diff
@@ -20,8 +20,10 @@ pysolr==3.9.0
 qtconsole==5.1.1
 retrying==1.3.3
 # pretrained coreference model dependency. Will load latest supported version of spacy, and transformers
-https://github.com/explosion/spacy-experimental/releases/download/v0.6.1/en_coreference_web_trf-3.4.0a2-py3-none-any.whl
-sentence-transformers==2.2.2
+#https://github.com/explosion/spacy-experimental/releases/download/v0.6.1/en_coreference_web_trf-3.4.0a2-py3-none-any.whl
+spacy
+faiss-cpu
+sentence-transformers==3.0.1
```

Comment:
@dcrouch26 - can you please confirm that you tested this upgrade and that it is safe / doesn't break anything? I just wanted to confirmation that this change was both intentional and well tested before I merge this PR into master. Thanks!